test/resources/unit/mrd/test_srsnv_plotting_utils/ppmSeq_legacy_v5_333_LuNgs_08.params.json

```diff
@@ -31,15 +31,15 @@
     "next_3bp"
   ],
   "balanced_sampling_info_fields": null,
-  "hom_snv_featuremap": "/cromwell_root/cromwell-backend-ultima-data-307918/cromwell-execution/SubstitutionAnalysis/64721496-75c8-4c98-b4ba-970a8148342a/call-CreateHomSnvFeatureMap/balanced_ePCR_LA5_LA6_333_LuNgs_08.hom_snv.vcf.gz",
-  "single_substitution_featuremap": "/cromwell_root/cromwell-backend-ultima-data-307918/cromwell-execution/SubstitutionAnalysis/6ceac660-9358-44c1-a22f-7d324ac4dc9a/call-FeatureMap/FeatureMap/4adcfaa6-a0af-43e4-aa89-f29a7109b390/call-FeatureMapSingleSubstitutionsMerge/balanced_ePCR_LA5_LA6_333_LuNgs_08.featuremap.single_substitutions.vcf.gz",
+  "hom_snv_featuremap": "/path/to/ppmSeq_333_LuNgs_08.hom_snv.vcf.gz",
```

Comment:
/path/to/ ?
is it a place holder?

Reply:
Yes, it's an internal parameters that is part of the json files used for testing, none of these paths are valid in any case, it makes sure it can read the file and parse the numbers